components/data-table/__examples__/fixed-header.jsx

```diff
@@ -204,9 +204,10 @@ class Example extends React.Component {
 			<div
 				style={{
 					height: '200px',
-					width: '100%',
+					width: '800px',
 				}}
 			>
+				<style dangerouslySetInnerHTML={{ __html: '.dsr_data-table-fixed-header-scroll-test { width: 1600px; }' }} />
```

Comment:
@interactivellama this was added for testing... do we want to leave this stuff in there and make a vertical / horizontal fixed header story example the default that shows up on the site? Or should I make another example with horizontal scrolling in order to preserve the simpler vertical-only scrolling fixed header example?

Reply:
I'd lean toward another story, although if we can just pass in a prop that modifies the same story, that'd probably be best.

Reply:
@kevinparkerson: create a `style` prop that gets applied to the table. Create a separate fixed header horizontal story that demonstrates horizontal scrolling without adding it to doc site. Update doc site to refer to using `style` prop to obtain horizontal scrolling (have a note in `style` prop description and a note in the top description of the component referencing the `style` prop)